src/languages/en-US.ts

```diff
@@ -594,6 +594,14 @@ export default class extends Language {
 					game in which the players first choose a color and then take turns dropping colored discs from the top into a
 					seven-column, ~~six~~ five-row vertically suspended grid.`
 		}),
+		COMMAND_COINFLIP_DESCRIPTION: 'Flip a coin!',
+		COMMAND_COINFLIP_EXTENDED: builder.display('coinflip', {
+			extendedHelp: `Flip a coin. If you guess the side that shows up, you get back your wager, doubled.
+				If you don't, you lose your wager.
+				You can also run a cashless flip, which doesn't cost anything, but also doesn't reward you with anything.
+				Now get those coins flippin'.`,
```

Comment:
```ts
extendedHelp: [
    'Flip a coin. If you guess the side that shows up, you get back your wager, doubled.',
    'If you don\'t, you lose your wager.',
    'You can also run a cashless flip, which doesn\'t cost anything, but also doesn\'t reward you with anything.',
    'Now get those coins flippin\'.'
].join('\n'),
```

Reply:
It's fine in the builder, it automatically trims indentation.